docs/hardware.md

```diff
@@ -1,91 +1,162 @@
+#### Workstations
 
- * 1 x laptop without a hard drive, network interface card or wireless units.
- * 1 x encrypted, external hard drive to store documents on while working on a story.
- * 1 x offline printer.
+THese components are necessary to do the initial installation of SecureDrop and to process submissions using the airgapped workflow.
 
-We recommend that you either buy or repurpose an old laptop. Another option is to buy an [Intel NUC](http://www.amazon.com/dp/B00F3F38O2/) with a power cord and [4 GB of memory](http://www.amazon.com/Crucial-PC3-12800-204-Pin-Notebook-CT2CP25664BF160B/dp/B005MWQ6WC/), but note that you will also need to get a monitor and a wired keyboard and mouse. For the printer, we recommend the [HP Deskjet printer](http://www.amazon.com/HP-Deskjet-Printer-CH340A-B1H/dp/B003YGZIY0/).
+- **Secure Viewing Station (SVS)**: 1 physical computer used as an airgap to decrypt and view submissions retrieved from the **Application Server**.
+    - The chosen hardware should be solely used for this purpose and should have any wireless networking hardware removed before use.
+- **Admin/Journalist Workstation(s)**: *At least 1* physical computer that is used as a workstation for SecureDrop admins and/or journalists.
+    - Each Admin and Journalist will have their own bootable Tails USB with an encrypted persistent partition that they will use to access SecureDrop. You will need at least one *workstation* to boot the Tails USBs, and may need more depending on: the number of admins/journalists you wish to grant access to SecureDrop, whether they can share the same workstation due to availability requirements, geographic distribution, etc.
+- **USB drive(s)**: *At least 2* USB drives to use as a bootable Tails USB for the **SVS** and the **Admin Tails**/**Journalist Tails**.
+    - If only one person is maintaining the system, you may use the same Tails instance as both the Admin Tails and the Journalist Tails; otherwise, we recommend buying 1 drive for each admin and each journalist.
+    - We also recommend buying two additional USBs to use as bootable backups of the **SVS** and **Admin Tails**.
+- **Two-factor authenticator**: Two-factor authentication is used when connecting to different parts of the SecureDrop system. Each admin and each journalist needs a two-factor authenticator. We currently support two options for two-factor authentication:
+    - Your existing smartphone with an app that computes TOTP codes (e.g. [Google Authenticator][])
+    - A dedicated hardware dongle that computes HOTP codes (e.g. a [Yubikey][]).
```

Comment:
Nit: `YubiKey` should have two capital letters, as it appears in Yubico's documentation, and elsewhere in the hardware guide.